src/Tests/UnitTests/Api/ApiTestUtil.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.IO;
+using System.Text;
+
+namespace UnitTests.Api
+{
+    public class ApiTestUtil
+    {
+        internal static string ReadDataFile(string name)
+        {
+            string path = Directory.GetCurrentDirectory();
+            string fileSpec = path + "\\..\\..\\..\\Api\\Data\\" + name;
+            return File.ReadAllText(fileSpec, Encoding.ASCII);
```

Comment:
Why explicitly ASCII?
```suggestion
            return File.ReadAllText(fileSpec);
```

Reply:
Just used to java...